products/knownprojects/sql/aggregate/_aggregate_longform_models.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+
+models:
+  - name: longform_nta_output
+    description: "Project records split by intersecting NTAs"
```

Comment:
Are these accurate descriptions? I'm not quite sure what this means

Reply:
yup their accurate. I'll revise them for clarity, so maybe something like `Project records with a row for each NTA it intersects with`

expand/explode/unnest are relevant concepts but an unnest isn't used in the queries and they don't seem to work well in a table description for non-SQL ppl

Reply:
Ah gotcha. Thanks, that description is clearer!